package.json

```diff
@@ -107,9 +107,9 @@
     "json-loader": "^0.5.7",
     "mini-css-extract-plugin": "^2.5.3",
     "minimatch": "^3.0.4",
-    "node-sass": "^4.14.1",
+    "sass": "^1.49.9",
+    "sass-loader": "^12.6.0",
     "promise-polyfill": "^8.1.3",
-    "sass-loader": "^12.4.0",
```

Comment:
why the sass changes?

Reply:
The node-sass library has been deprecated for over a year and actually requires additional software to run.  The sass package is its predecessor.  There should be no differences other than "sass" package has new and up to date features from the lang itself, requires no third party tooling/downloads (at least for windows), and is maintained through node versions.

You can see their warning and suggestion below.
https://github.com/sass/node-sass

Reply:
cool, and thanks, but our practice is that changes unrelated to what is being addressed by a PR should be done in a separate PR.  Can you please create a ticket for this and put it in the Triage column (the default "pipeline") of the kanban, and back the change out of this PR?